jobs/collect_logs.sh

```diff
@@ -80,13 +80,19 @@ echo "Extracting the logs tar archive"
 ssh -o "UserKnownHostsFile /dev/null" -o "StrictHostKeyChecking no" -i $LOGS_SSH_KEY logs@logs.openstack.tld "tar -xzf $LOGSDEST/aggregate-logs.tar.gz -C $LOGSDEST/"
 
 #echo "Uploading threaded logs"
-#set +e
-#scp -o "UserKnownHostsFile /dev/null" -o "StrictHostKeyChecking no" -i $LOGS_SSH_KEY /home/jenkins-slave/logs/devstack-build-log-$JOB_TYPE-$ZUUL_UUID logs@logs.openstack.tld:$LOGSDEST/
-#scp -o "UserKnownHostsFile /dev/null" -o "StrictHostKeyChecking no" -i $LOGS_SSH_KEY /home/jenkins-slave/logs/cinder-windows-build-log-$JOB_TYPE-$ZUUL_UUID logs@logs.openstack.tld:$LOGSDEST/
-#set -e
+set +e
+scp -o "UserKnownHostsFile /dev/null" -o "StrictHostKeyChecking no" -i $LOGS_SSH_KEY /home/jenkins-slave/logs/hyperv-$hyperv01-build-log-$ZUUL_UUID-$JOB_TYPE logs@logs.openstack.tld:$LOGSDEST/
```

Comment:
These should all be gzipped before scp and they should also use .log extension (when redirecting). For sample, see https://github.com/cloudbase/nova-ci/commit/f7b300cca1646b4b927e8f1dc022a2fab76d6312.